### DISCUSSION TITLE
Is positron a fork of vs code or codium and how much influence does microsoft have?

### DISCUSSION THREAD

**Discussion by SergejRuff** [posit-dev/positron#5347] [Q&A]
Hi, I'm currently an RStudio user considering switching to Positron for simple data analysis in Python and R, while continuing to use RStudio for more complex tasks, like R package development. However, I’m concerned about Microsoft products and their tracking practices.

Could you clarify if Positron is a fork of VS Code or Codium? If it’s closer to VS Code, how much influence does Microsoft have over it? Thanks!

**Comment by DavisVaughan** [comment]
Positron is _technically_ a fork of Code OSS

See this on the VS Code README:
https://github.com/microsoft/vscode

> [Visual Studio Code](https://code.visualstudio.com/) is a distribution of the Code - OSS repository with Microsoft-specific customizations released under a traditional [Microsoft product license](https://code.visualstudio.com/License/).

Code OSS itself is an MIT licensed open source product that Microsoft develops.

Regarding influence, they don't have any influence over our day to day development, but we resync with upstream Code OSS once a month, so we inherit all of their bug fixes and new features.

---

Regarding your use case, you might be interested to know that a number of people on the r-lib/tidyverse package development team work on Positron, meaning that we have honestly hyper optimized it for package development even more than we have for data analysis so far! I do all of my R package development in Positron these days, and don't feel like I'm missing anything from RStudio at this point.

**Reply by SergejRuff** [reply]
Thank you for your reply. So, there is no telemetry or tracking of user data without the knowledge of the user, right?

**Reply by DavisVaughan** [reply]
See https://github.com/posit-dev/positron/discussions/5160#discussioncomment-11044468
